src/hooks/hooks.js

```diff
@@ -0,0 +1,15 @@
+import { useState, useEffect } from 'react';
```

Comment:
nitpick: Why is this at `hooks/hooks.js`? It could be in a better place. eg `state/useLocaStorage.js` to be more specific as the app grows :)

Reply:
I thought it should be in a separate file and this was what I could come up with:D I will change it to state/ useLocalStrogae.js